cmd/urunc/delete.go

```diff
@@ -34,27 +35,28 @@ For example, if the container id is "ubuntu01" and runc list currently shows the
 status of "ubuntu01" as "stopped" the following will delete resources held for
 "ubuntu01" removing "ubuntu01" from the runc list of containers:
 
-       # runc delete ubuntu01`,
+	# runc delete ubuntu01`,
```

Comment:
Could you also replace `runc` with `urunc`. For some reason, we never changed it...